tsconfig.json

```diff
@@ -20,9 +17,7 @@
     "isolatedModules": true,
     "baseUrl": ".",
     "paths": {
-      "@/*": [
-        "src/*"
-      ]
+      "@/*": ["src/*"]
```

Comment:
they are just formatting changes not a part of the package upgrade to refactor.

Reply:
But that is the correct formatting right? Then, why not just add it?

Reply:
lets say that is the formatting in your system in my system prettier runs in some different way then?

Reply:
This is the formatting specified for our project